Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10.14.1-alpine
```

Comment:
se tu for usar sass, **acho** q vai dar ruim, usar alpine

Reply:
deu bom, hue